[STORY-678] feat(db): Add new db dashboard documentation [PG PART] (#2847)
Scalingo/documentation: src/_posts/databases/postgresql/2000-01-01-managing.md

2. Click on the application for which you want to scale the Scalingo for
   PostgreSQL® addon
3. Click on the **Resources** tab
4. Locate the **Addons** block
5. Click the **"&#8230;"** button
6. From the dropdown menu, select **Change plan**
7. Select the new plan

> 8. Validate by clicking the **Finish** button

Reviewer: Sorry this one is my fault :-(
```suggestion
8. Click the **Finish** button
```

2. Click on the application for which you want to scale the Scalingo for
   PostgreSQL® addon
3. Click on the **Resources** tab

> 4. Locate the **Addons** block and click on the **"&#8230;"** button
> 5. From the dropdown menu, select **Change plan**
> 6. Select the new plan
> 7. Click the **Finish** button

Reviewer: ```suggestion
4. Locate the **Addons** block
5. Click the **"&#8230;"** button
6. From the dropdown menu, select **Change plan**
7. Select the new plan
8. Validate by clicking the **Finish** button
```

### Using the Database Dashboard

1. From your web browser, [open your database dashboard]({% post_url databases/postgresql/2000-01-01-getting-started %}#accessing-the-scalingo-for-postgresql-dashboard)
2. Click on the **Settings** tab. Locate the **Database Upgrade** block
3. In **General**, if an upgrade is available, the text in the block explains what will be

> 4. To launch the upgrade, click the **Upgrade to ...** button

Reviewer: ```suggestion
2. Select the **Settings** tab
3. In the **Settings** submenu, select the **General** tab
4. Locate the **Database Version** block
5. If an upgrade is available, a button allows you to trigger the upgrade
6. Click the button to launch the upgrade process
```

#### Using the Database Dashboard

1. From your web browser, [open your database dashboard]({% post_url databases/postgresql/2000-01-01-getting-started %}#accessing-the-postgresql-dashboard)
2. Click on the **Settings** tab
3. In the **Settings** menu, select **Users**

> 4. The list of available users is displayed under the **User Management**

Reviewer: ```suggestion
2. Click the **Settings** tab
3. In the **Settings** submenu, select **Users**
4. The list of available users is displayed in the **User Management** block
```

Author: already implemented

#### Using the Database Dashboard

1. From your web browser, [open your database dashboard]({% post_url databases/postgresql/2000-01-01-getting-started %}#accessing-the-scalingo-for-postgresql-dashboard)
2. Click on the **Settings** tab

> 3. In the **Settings** menu, select **Users**

Reviewer: ```suggestion
2. Click the **Settings** tab
3. In the **Settings** submenu, select **Users**
```

Author: already implemented

#### Using the Database Dashboard

1. From your web browser, [open your database dashboard]({% post_url databases/postgresql/2000-01-01-getting-started %}#accessing-the-scalingo-for-postgresql-dashboard)
2. Click the **Settings** tab
3. In the **Settings** submenu, select **Users**
4. Fill the **Add a user** form:
   - Fill a username

> 5. (optional) If you want to allow this user to have writings abilities, make
>    sure to check the **Write** checkbox
> 6. Validate by clicking the **Add this user** button

Reviewer: ```suggestion
   - (optional) To allow the new user to have writings abilities, make
     sure to check the **Write (optional)** checkbox
5. Validate by clicking the **Add this user** button
```

#### Using the Database Dashboard

1. From your web browser, [open your database dashboard]({% post_url databases/postgresql/2000-01-01-getting-started %}#accessing-the-scalingo-for-postgresql-dashboard)
2. Click the **Settings** tab
3. In the **Settings** submenu, select **Users**
4. Fill the **Add a user** form:
   - Fill a username
   - (optional) To allow the new user to have writings abilities, make
     sure to check the **Write (optional)** checkbox
5. Validate by clicking the **Add this user** button

> {% note %}
> A secure password will be generated automatically. You will be able to view and copy it only once.
> It will appear in a popup window after the user creation.
> {% endnote %}

Reviewer: ```suggestion
{% note %}
A secure password is generated automatically and shown in a popup window after the user creation.
You will be able to view and copy it only once.
{% endnote %}
```

Author: Implemented

### Updating a User Password

{% note %}
Updating a User Password is only available from the command line.

> Protected users password cannot be updated

Reviewer: ```suggestion
Protected users password cannot be updated.
```

#### Using the Database Dashboard

1. From your web browser, [open your database dashboard]({% post_url databases/postgresql/2000-01-01-getting-started %}#accessing-the-scalingo-for-postgresql-dashboard)
2. Click on the **Settings** tab
3. In the **Settings** menu, select **Users**
4. Locate the user you want to remove
5. Click the **"..."** button next to the user and select **Delete user**

> 6. Confirm the deletion by typing the name of the user in the popup window

Reviewer: ```suggestion
2. Click the **Settings** tab
3. In the **Settings** submenu, select **Users**
4. Locate the user you want to remove
5. Click the **…** button next to the user and select **Delete user** from the popup menu
6. In the popup window, confirm the deletion by typing the name of the user
```

Author: already implemented

   User "my_user" created with password "YANs3y07m5_KJC2MSDGebh8tx1lliFWh2Yb239zVqGQvbElWDjIN7QWspVH92Ul8".
   ```

### Deleting a User

{% note %}

> Protected users cannot be deleted

Reviewer: ```suggestion
Protected users cannot be deleted.
```

{% endnote %}

#### Using the Database Dashboard

1. From your web browser, [open your database dashboard]({% post_url databases/postgresql/2000-01-01-getting-started %}#accessing-the-scalingo-for-postgresql-dashboard)
2. Click the **Settings** tab
3. In the **Settings** submenu, select **Users**
4. Locate the user you want to remove

> 5. Click the **…** button next to the user and select **Delete user** from the popup menu
> 6. In the popup window, confirm the deletion by typing the name of the user
> 7. Validate by clicking the **Confirm** button

Reviewer: ```suggestion
5. Click the **&#8230;** button next to the user
6. From the popup menu, select **Delete user**
7. In the popup window, confirm the deletion by typing the name of the user
8. Validate by clicking the **Confirm** button
```

Please refer to the Overview page to get the [list of available extensions]({% post_url databases/postgresql/2000-01-01-overview %}#available-extensions).

> ### Show the enabled extensions

Reviewer: ```suggestion
### Viewing Enabled Extensions
```

#### Using the Database Dashboard

1. From your web browser, [open your database dashboard]({% post_url databases/postgresql/2000-01-01-getting-started %}#accessing-the-scalingo-for-postgresql-dashboard)
2. Click on the **Settings** tab
3. In the **Settings** menu, select **Condiguration**
4. Locate the **PostgreSQL Extensions**

> 5. The list of enabled extensions will appear in small pills

Reviewer: ```suggestion
2. Click the **Settings** tab
3. In the **Settings** submenu, select **Configuration**
4. Locate the **PostgreSQL Extensions** block
5. The list of enabled extensions is displayed here
```

Author: already implemented

Please refer to the Overview page to get the [list of available extensions]({% post_url databases/postgresql/2000-01-01-overview %}#available-extensions).

### Viewing Enabled Extensions

#### Using the Database Dashboard

1. From your web browser, [open your database dashboard]({% post_url databases/postgresql/2000-01-01-getting-started %}#accessing-the-scalingo-for-postgresql-dashboard)
2. Click the **Settings** tab
3. In the **Settings** submenu, select **Configuration**

> 4. Locate the **PostgreSQL Extensions** block
> 5. The list of enabled extensions is displayed here

Reviewer: ```suggestion
4.  The list of enabled extensions is displayed in the **PostgreSQL Extensions** block
```
